### PR TITLE
Top navbar and side navbar

### PR DESCRIPTION
**Pull request by OmriGilhar**
Change base.html file replacing navbar to bootstrap and adding a sidebar
tow of the changes added without any functionality.

### CODE REVIEW

File: twitch_rivals/templates/base.html [hunk 1]
```diff
@@ -1,15 +1,96 @@
...
-            <h1>RivalCast</h1>
-    </div>
+
+    <meta charset="utf-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content=""/>
+    <meta name="author" content=""/>
+
+    <title>Starter Template for Bootstrap</title>
```
Comment (danielBelenky) [inline twitch_rivals/templates/base.html]: Maybe add a title here

File: twitch_rivals/templates/base.html [hunk 2]
```diff
@@ -1,15 +1,96 @@
...
+
+    <!-- Bootstrap Core CSS -->
+    <link rel="stylesheet" href="https://mdbootstrap.com/wp-content/themes/mdbootstrap4/css/compiled-4.5.14.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/dragula/3.7.2/dragula.min.css">
+    <link rel="stylesheet" type="text/css" href="https://mdbootstrap.com/wp-content/themes/mdbootstrap4/css/mdb-plugins-gathered.min.css">
+    <script src= "https://player.twitch.tv/js/embed/v1.js"></script>
+    <!-- Custom CSS -->
+    <link rel="stylesheet" href="{{ url_for('static', filename='css/base.css') }}">
```
Comment (danielBelenky) [inline twitch_rivals/templates/base.html:15]: can this be static instead of being a jinja template? we know the path ...

**Comment by danielBelenky** [comment]
Can you fix the commit message? The title is too long.
I recommend following this blog post: https://chris.beams.io/posts/git-commit/

Also, you specify that you do not add any functionality, but have changed the UI...

**Review by danielBelenky — CHANGES_REQUESTED** [review]

File: twitch_rivals/templates/base.html (hunk 2 above)
Comment (lkashti) [reply twitch_rivals/templates/base.html:15]: Solution may be here
https://stackoverflow.com/questions/25034812/use-a-css-stylesheet-on-a-jinja2-template

File: twitch_rivals/templates/base.html (hunk 1 above)
Comment (OmriGilhar) [reply twitch_rivals/templates/base.html]: I removed the meta and the title

File: twitch_rivals/templates/base.html (hunk 2 above)
Comment (OmriGilhar) [reply twitch_rivals/templates/base.html:15]: @lkashti sent a StackOverflow answer that shows the current solution, I think we need to stick with the current code, maybe we need to change the design for the pages.

**Comment by danielBelenky** [comment]
:+1:

**Review by danielBelenky — APPROVED** [review]

**Pull request merged**